### PR TITLE
add the reset-form-function

### PR DESCRIPTION
**Pull request by martinhaiss**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -189,24 +190,25 @@ <h2 class="card__question">Woher kommt der Name "Frisbee"?</h2>
...
+            placeholder="Hier deine Antwort geben"
+          ></textarea>
+          <label for="tags">Tags durch Komma trennen</label><br />
+          <input type="text" id="tags" name="tags" /><br />
+          <input
+            type="button"
+            onclick="reset()"
+            class="submit"
+            value="submit"
+          />
```
Comment (macfrei) [inline index.html:211]: Let's not write JavaScript directly in our HTML 🙂

File: create.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+// reset input-fields of new card
+
+
+function cardReset() {
+  document.getElementById('#card__create').reset()
+  console.log(Hello form-reset!);
+}
```
Comment (macfrei) [inline create.js:7]: Have a look at this codepen, at my HTML and JavaScript: https://codepen.io/macfrei/pen/mdwOEBd?editors=0010

I know we haven't discussed this in the course yet, but I will give you the solution now anyways 😉: 

1. The `form` element in HTML has a `submit` event we can use
2. We need to select the `form` 
3. The `form` gets an event listener `addEventListener('sumbit', (event) => {...})`
4. We also get an `event` object which we can use
5. Now the `form` can be reset: `form.reset()`

If you have any questions, please get back to me!

**Review by macfrei — COMMENTED** [review]
Well done! Added a few comments 👍

File: create.js (hunk 2 above)
Comment (martinhaiss) [reply create.js:7]: Cool, thx! I'll implement this 🤓

**Pull request merged**